crates/primitives/src/result.rs

```diff
@@ -217,6 +218,68 @@ pub enum InvalidTransaction {
     DepositSystemTxPostRegolith,
 }
 
+impl error::Error for InvalidTransaction {}
```

Comment:
You need to feature gate this with: `#[cfg(feature = "std")]` and use `std::error::Error` as `core::error::Error` is still experimental

https://doc.rust-lang.org/std/error/trait.Error.html